practice-two/src/styles/layouts/comment.scss

```diff
@@ -0,0 +1,88 @@
+.comment {
+  margin: toRem(15px) 0 toRem(50px) 0;
+  font-family: Satoshi;
+  overflow: hidden;
+
+  &__card {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    gap: toRem(20px);
+  }
+
+  .comment__head {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+  }
+
+  &__item {
+      margin-top: toRem(24px);
+      padding: toRem(24px);
+      min-width: toRem(358px);
+      border: 1px solid $line-color;
+      border-radius: toRem(20px);
+  }
+
+  &__evaluate {
+    min-width: toRem(118px);
+
+    &-img {
+      max-width: toRem(19.19px);
+      min-height: toRem(19.19px);
+    }
+  }
+
+  &__name {
+    display: flex;
+    align-items: center;
```

Comment:
@thanhtrantruong47 this two properties can reusable by using a placeholder in sass